package.json

```
@@ -43,7 +43,7 @@
     "remove-types": "^1.0.0",
     "serve": "^13.0.2",
     "typedoc": "^0.23.2",
-    "typescript": "4.8.0-dev.20220627",
```

typedoc declares typescript as a peer, and does not include `dev` releases

---

Is this actually ok?

---

Can we just tell pnpm to ignore the error?

---

that's an option, yeah -- I'll switch to that since we're using 4.8-dev everywhere